mycroft/stt/__init__.py

```diff
@@ -22,6 +22,12 @@
 from mycroft.configuration import ConfigurationManager
 from mycroft.util.log import getLogger
 
+
+#imports for json handling, requests and string formatting for KaldiSTT
```

Comment:
Sorry about the confusion. The configuration should use the `self` operator. That way, you'll have access to the module configuration itself. Please, do the following update:
```
    def execute(self, audio, language=None):
        language = language or self.lang
        response = requests.post(self.config.get("uri"), data=audio.get_wav_data())        
        return get_response(response)
```
Then, your config should be like: 
```
  "stt": {
    "module": "kaldi",
    "kaldi": {
      "uri": "http://localhost:8081/client/dynamic/recognize"
    }
  }
```
To rebase you just have to update your local `dev` branch based on our `upstream`. Then, rebase your feature on it:
```
# make sure you have your upstream setup
git remote add upstream git@github.com:MycroftAI/mycroft-core.git

# update your local dev branch based on our upstream
git fetch upstream
git checkout dev
git reset --hard upstream/dev

# rebase your feature branch based on dev and force the push to remote
git checkout feature/issues-438
git rebase dev
git push -f
```